xml/System.Runtime.Loader/AssemblyLoadContext.xml

```diff
@@ -189,10 +237,33 @@
         <Parameter Name="assemblyName" Type="System.Reflection.AssemblyName" />
       </Parameters>
       <Docs>
-        <param name="assemblyName">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="assemblyName">The object that describes the assembly to be loaded.</param>
+        <summary>Resolves and loads an assembly given its <see cref="T:System.Reflection.AssemblyName" />.</summary>
+        <returns>The loaded assembly.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
```

Comment:
Suggestion: It's worth mentioning here that different versions of the same assembly cannot be loaded into the same ALC, but can be loaded into different ALCs.